docs/bundle.js

```diff
@@ -69779,6 +69779,7 @@
 	    className: PropTypes.string,
 	    error: PropTypes.string,
 	    size: PropTypes.oneOf(Object.values(Forms_1.FormElementSize)),
+	    dontCloseMenuOnSelect: PropTypes.bool,

```

Comment:
Hmm, yeah after doing a little thinking, I think I'd prefer `closeMenuOnSelect`. My reasoning is:
* It becomes more straightforward for Clever developers to mentally map this prop to the React Select prop
* It's generally easier to reason about things when they are positively named (see for example this blog post on [avoiding negative variable names](http://davej.io/2018/05/negative-naming.html))

But those aren't super strong opinions. What do you think?

Reply:
I think you meant to comment on the diff in `Select.tsx` 😛

Reply:
Lol yeah.. I read the PR description, had a comment and quickly found a place to drop it 😆 this was a pretty random place

Reply:
I also prefer positively named booleans. I can see a compelling case for either, but you swayed me in the direction of `closeMenuOnSelect`.